support/gathering-cluster-data.adoc

```diff
@@ -38,14 +38,16 @@ include::modules/gathering-data-specific-features.adoc[leveloffset=+2]
 
 ifndef::openshift-rosa,openshift-dedicated[]
 * xref:../nodes/cma/nodes-cma-autoscaling-custom.adoc#nodes-cma-autoscaling-custom-gather[Gathering debugging data] for the Custom Metrics Autoscaler.
+* link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy]
 endif::openshift-rosa,openshift-dedicated[]
 
-* link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy]
+ifdef::openshift-rosa[]
+* xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.html[{product-title} update life cycle]
+endif::openshift-rosa[]
 
-ifdef::openshift-rosa,openshift-dedicated[]
-// Gathering audit logs
-include::modules/gathering-data-audit-logs.adoc[leveloffset=+2]
```

Comment:
@michaelryanmcneill @bmcelvee I think this got unintentionally omitted - it doesn't seem to be either excluded from or included in openshift-dedicated or openshift-rosa builds at the moment. Hard to tell because the assembly it is included in was previously commented out of the topic map.

Reply:
This is intentionally removed as we properly included the audit log documentation in #75292. In addition, a module on the page already includes the same information from the omitted module.